### PR TITLE
test: add unit tests for uploads

### PR DESCRIPTION
**Pull request by thaaaarun**
Fixes #<9>

> It's a good idea to open an issue first for discussion.

- [/ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: front_end/test/common.js
Comment (ddelgrosso1) [inline front_end/test/common.js:27]: Not sure if you are linting tests yet but this line looks long. Probably should be split.

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:96]: can this be `const`?

File: front_end/test/downloads.js
Comment (shaffeeullah) [inline front_end/test/downloads.js:41]: can this be `const`?

File: front_end/test/downloads.js
Comment (shaffeeullah) [inline front_end/test/downloads.js:106]: `const`

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:52]: `const`

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:102]: const

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:170]: const

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:185]: const

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:206]: const

File: front_end/test/uploads.js
Comment (shaffeeullah) [inline front_end/test/uploads.js:221]: const

**Review by shaffeeullah — COMMENTED** [review]
A few nit comments about `let` vs `const`. Otherwise, looks good! Your PRs are getting much much cleaner. Awesome job, Tharun!!

File: front_end/test/uploads.js
Comment (ddelgrosso1) [inline front_end/test/uploads.js:67]: Consider making the error its own variable.

**Review by shaffeeullah — APPROVED** [review]
LGTM. Please wait for explicit approval from @ddelgrosso1 before merging.

**Pull request merged**
